parts/chapters/subsections/10.3/RPTRST.fodt

```diff
@@ -4478,184 +4468,434 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
        <text:p text:style-name="P4938">ALLPROPS</text:p>
       </table:table-cell>
       <table:table-cell table:style-name="Table716.A2" office:value-type="string">
-       <text:p text:style-name="P4936">An alias for DEN, KRG, KRO, KRW, and VISC restart variable names combined that writes all the properties associated with these keywords.</text:p>
+       <text:p text:style-name="P4936">An alias for the BG, BO, BW, DEN, KRG, KRO, KRW, VGAS, VOIL and VWAT mnemonics combined that writes all the associated properties.</text:p>
       </table:table-cell>
       <table:table-cell table:style-name="Table716.D2" office:value-type="string">
        <text:p text:style-name="P4942"/>
       </table:table-cell>
      </table:table-row>
      <table:table-row table:style-name="Table716.1">
-      <table:table-cell table:style-name="Table716.A2" office:value-type="string">
-       <text:p text:style-name="P4944">2</text:p>
+      <table:table-cell table:style-name="Table716.A5" table:formula="ooow:&lt;A2&gt;+1" office:value-type="float" office:value="2">
+       <text:p text:style-name="P5428">2</text:p>
       </table:table-cell>
       <table:table-cell table:style-name="Table716.A2" office:value-type="string">
        <text:p text:style-name="P4945">BASIC</text:p>
       </table:table-cell>
       <table:table-cell table:style-name="Table716.A2" office:value-type="string">
-       <text:p text:style-name="P4945">BASIC defines the frequency at which the restart data for restarting a run and the additional requested data is written to the RESTART file. The parameter is assigned a value, OPTION, using the form BASIC = OPTION, where OPTION is an integer variable set to: </text:p>
+       <text:p text:style-name="Table_20_Contents">The BASIC mnemonic defines the frequency at which the restart data and the additional requested data is written to the RESTART file. The mnemonic is assigned a value, using the form BASIC=OPTION, where OPTION is an integer variable set to one of the following: </text:p>
        <text:list xml:id="list105919273011646" text:continue-list="list105918417837053" text:style-name="_40_NumberedListTable">
         <text:list-item text:start-value="1">
          <text:p text:style-name="P19573">OPTION = 0 then output to the restart file is stopped. </text:p>
         </text:list-item>
         <text:list-item>
-         <text:p text:style-name="P19573">OPTION = 1 then the restart files are written at every report time, but only the last one in the run is kept. This minimizes the restart file size but only the final results are stored, limiting the visualization in OPM ResInsight.</text:p>
+         <text:p text:style-name="P19573">OPTION = 1 then the restart files are written at every report time, but only the last one output is kept. This minimizes the restart file size but limits the visualization in OPM ResInsight to the final conditions.</text:p>
```

Comment:
"last one ouput is" reads strange: Either "last output is" or "last one is"

Reply:
Agreed, I have gone with "last output is"